### PR TITLE
Dockerfiles

### PR DESCRIPTION
**Pull request by rouazana**
#2 Move dockerfiles in James-project
Should also modify compile.sh script in order to accept a SHA1 instead of a branch name. (optional ? -> trunk)

### CODE REVIEW

**Comment by mbaechler** [comment]
:+1:

File: dockerfiles/compilation/java-8/compile.sh [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+fi
+
+# Sources retrieval
+git clone $ORIGIN/.
+git checkout $SHA1
+
+# Compilation
+
+if [ "$SKIPTESTS" = "skipTests" ]; then
+   mvn clean install -DskipTests -Pcassandra,exclude-lucene,with-assembly,with-jetm
```
Comment (mbaechler) [inline dockerfiles/compilation/java-8/compile.sh]: We need to change this to mvn package (clean doesn't make sense in my opinion)

**Comment by aduprat** [comment]
:+1:

**Comment by mbaechler** [comment]
:+1:

**Comment by aduprat** [comment]
:+1:

**Comment by mbaechler** [comment]
merged

**Pull request closed without merging**